src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const { name, age = null, sex, isMarried, partnerName } = person;
+  const married = sex === 'f' ? 'husband' : 'wife';
+  const merriedText = `${partnerName} is my ${merried}`;
```

Comment:
You should fix linter errors
```suggestion
  const merriedText = `${partnerName} is my ${married}`;
```
<img width="788" alt="image" src="https://user-images.githubusercontent.com/99403198/234532783-ad4fc465-2b1f-4858-8a94-05e500cb7a45.png">